Sources/Brave/Shortcuts/ActivityShortcutManager.swift

```diff
@@ -215,8 +215,7 @@ public class ActivityShortcutManager: NSObject {
   // MARK: Intent Donation Methods
 
   public func donateCustomIntent(for type: IntentType, with urlString: String) {
-    guard !PrivateBrowsingManager.shared.isPrivateBrowsing,
-          !urlString.isEmpty,
+    guard !urlString.isEmpty,
```

Comment:
Do we not need to check for private browsing here anymore?

Reply:
> Do we not need to check for private browsing here anymore?

All of the code inside of the intents already does. This code was duplicated, so I removed it.

So now it checks if private mode inside the function, and does nothing, instead of checking private mode twice.